packages/pn-pa-webapp/src/redux/auth/types.ts

```diff
@@ -7,10 +7,18 @@ export interface User extends BasicUser {
 }
 
 export interface Organization {
+  parentDescription?: string;
```

Comment:
For security reasons all the new keys must be validated here
packages/pn-pa-webapp/src/redux/auth/reducers.ts

Reply:
@AndreaCimini90 you can check the pr again